scipy/sparse/_compressed.py

```diff
@@ -897,8 +897,8 @@ def check_bounds(indices, bound):
                 raise IndexError('index (%d) out of range (< -%d)' %
                                  (idx, bound))
 
-        i = np.array(i, dtype=self.indices.dtype, copy=False, ndmin=1).ravel()
-        j = np.array(j, dtype=self.indices.dtype, copy=False, ndmin=1).ravel()
+        i = np.atleast_1d(np.asarray(i, dtype=self.indices.dtype)).ravel()
+        j = np.atleast_1d(np.asarray(j, dtype=self.indices.dtype)).ravel()
```

Comment:
Just a comment that this is correct (no need to do anything). It's slightly annoying to have the double function call, but `atleast_1d` uses `asanyarray` rather than `asarray` internally so this is the safe choice.